static/app/views/performance/browser/webVitals/components/performanceScoreRingWithTooltips.tsx

```diff
@@ -172,8 +174,11 @@ function PerformanceScoreRingWithTooltips({
   const [webVitalTooltip, setWebVitalTooltip] = useState<WebVitals | null>(null);
   const [labelHovered, setLabelHovered] = useState<WebVitals | null>(null);
 
+  const shouldReplaceFidWithInp = useReplaceFidWithInpSetting();
+  const ringSegmentOrder = shouldReplaceFidWithInp ? ORDER_WITH_INP : ORDER;
```

Comment:
At this point it might be cleaner to put this logic into a hook, and return correct orders and weights based on this setting, rather than having conditionals on `shouldReplaceFidWithInp` in a bunch of components